cgmes-gl-server/src/main/resources/application.properties

```diff
@@ -0,0 +1,5 @@
+server.port=8095
+
+geo-data-server.base.url=http://localhost:8087
+case-server.base.url=http://localhost:5000
```

Comment:
It changes nothing but why use a application.properties instead of application.yml ? Probably better to have the same config everywhere